packages/api/lib/granules.js

```diff
@@ -20,6 +21,51 @@ const translateGranule = async (
   };
 };
 
+const getExecutionProcessingTimeInfo = ({
+  startDate,
+  stopDate,
+  now = new Date(),
+}) => {
+  const processingTimeInfo = {};
+  if (startDate) {
+    processingTimeInfo.processingStartDateTime = startDate.toISOString();
+    processingTimeInfo.processingEndDateTime = stopDate
+      ? stopDate.toISOString()
+      : now.toISOString();
+  }
+  return processingTimeInfo;
+};
+
+/* eslint-disable camelcase */
+
+const getGranuleTimeToPreprocess = ({
+  sync_granule_duration = 0,
+} = {}) => sync_granule_duration / 1000;
+
+const getGranuleTimeToArchive = ({
+  post_to_cmr_duration = 0,
```

Comment:
We are? We're just default to `0` since all the function does is divide the incoming value by 1000, and `0` is the valid result even if the incoming value is undefined, I think